server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16-alpine
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY *.go ./
+RUN go build -o /server
+
+EXPOSE 8080
```

Comment:
could we switch a user after we install deps?

Reply:
done